sdk/proto/phpfpm.proto

```diff
@@ -0,0 +1,87 @@
+syntax = "proto3";
+package f5.nginx.agent.sdk;
+
+import "gogo.proto";
+
+option go_package = "github.com/nginx/agent/sdk/v2/proto;proto";
+
+// PhpFpmDetails registers a phpfpm master process + child pools
+message PhpFpmDetails {
+  // PhpFpm process type (e.g. phpfpm)
+  string type = 1 [(gogoproto.jsontag) = "type"];
+  // System ID
+  string uuid = 2 [(gogoproto.jsontag) = "uuid"];
+  // PhpFpm ID
+  string local_id = 3 [(gogoproto.jsontag) = "local_id"];
+  // PhpFpm process name
+  string name = 4 [(gogoproto.jsontag) = "name"];
+  // PhpFpm process descpription
+  // Example: php-fpm: master process (/etc/php/7.4/fpm/php-fpm.conf)
+  string cmd = 5 [(gogoproto.jsontag) = "cmd"];
```

Comment:
should this be process_path? https://github.com/nginx/agent/blob/main/sdk/proto/nginx.proto#L27

Reply:
no. should be `process_path`. I will rename `bin_path` instead.